Python/Python.sublime-syntax

```diff
@@ -171,7 +171,7 @@ contexts:
     - meta_include_prototype: false
     - meta_scope: comment.line.shebang.python
     # Note: Keep sync with first_line_match!
```

Comment:
Is there a reason not to use a variable?

Reply:
The comment might still be necessary :
https://github.com/sublimehq/sublime_text/issues/4713